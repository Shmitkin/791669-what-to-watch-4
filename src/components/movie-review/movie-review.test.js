import React from "react";
import renderer from "react-test-renderer";
import MovieReview from "./movie-review.jsx";

const review = {
  text: `review text`,
  author: `author`,
  date: `date string`,
  rating: 0.5,
};

it(`Should MovieReview render correctly`, () => {
  const tree = renderer
    .create(
        <MovieReview
          review = {review}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
