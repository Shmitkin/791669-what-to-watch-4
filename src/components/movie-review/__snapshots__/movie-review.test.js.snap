// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieReview render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      comment text
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        author1
      </cite>
      <time
        className="review__date"
        dateTime="2020-06-16T10:54:44.818Z"
      >
        June 16, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    7
  </div>
</div>
`;
